client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -23,13 +34,92 @@ const isMonthsOld = ( months: number, rawDate?: string ) => {
 	return moment().diff( parsedDate, 'months' ) > months;
 };
 
-export default function PlanOnlyThankYou( { primaryPurchase }: PlanOnlyThankYouProps ) {
+export default function PlanOnlyThankYou( {
+	primaryPurchase,
+	isEmailVerified,
+}: PlanOnlyThankYouProps ) {
 	const siteId = useSelector( getSelectedSiteId );
 	const siteSlug = useSelector( getSelectedSiteSlug );
+	const emailAddress = useSelector( getCurrentUserEmail );
```

Comment:
We typically declare constants/variables such that they're closer to the block of code that uses them, so it's obvious they are used for.